servicemesh-install/topics/appendix-threescale-adapter.adoc

```diff
@@ -21,7 +22,32 @@ You can use these examples to configure requests to your services using the 3sca
 Pay particular attention to the `kind: handler` resource. You must update this with your 3scale credentials and the service ID of the API you want to manage.
 ====
 
-Modify the handler configuration with your 3scale configuration.
+. Modify the handler configuration with your 3scale configuration.
++
+[source,yaml]
+----
+  apiVersion: "config.istio.io/v1alpha2"
+  kind: handler
+  metadata:
+   name: threescale
+   namespace:
+  spec:
+   adapter: threescale
+   params:
+     service_id: "<SERVICE_ID>"
+     system_url: "https://<organization>-admin.3scale.net/"
+     access_token: "<ACCESS_TOKEN>"
+   connection:
+     address: "threescale-istio-adapter:3333"
+----
+
++
+. Create the required resources:
++
+----
+oc create -f istio/ -n 
+----
```

Comment:
@dfennessy @PhilipGough I have removed the namespace label from the yaml example and the oc create command. Let me know if you're happy with this change.

Reply:
Should it be `oc create -f istio/ -n {replace-me}`

Reply:
Yes, needs a namespace provided as is or else to drop the `-n`. I prefer the former

Reply:
@geekspertise `namespace: ` line should be removed entirely

Reply:
@PhilipGough and @dfennessy All namespace references are now removed.